README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
+`--CanvPlotType` | string | Defines the type of `TObject` to be plotted on the `TCanvas`. Supported types are from the set {`TGraph`, `TGraph2D`, `TGraphErrors`, `TH1F`, `TH2F`}. Default is: TGraphErrors
+`--CanvRangeX` | Two comma separated int's | Defines the range `[X_min, X_max]` of the X-axis. Default is: 0,1000
+`--CanvRangeY` | Two comma separated int's | As `CanvRangeX` but for the Y-axis. Default is: 0,7
+`--CanvTitleOffsetX` | float | Defines the offset of the X-axis title. Default is: 1.0
+`--CanvTitleOffsetY` | float | As `CanvTitleOffsetX` but for the Y-axis. Default is: 0.8
+`--CanvTitleX` | string | Title (e.g. label) assigned to the X-axis.
+`--CanvTitleY` | string | As `CanvTitleX` but for the Y-axis.
+`--CanvName` | string| Set Canvas Name, Default is: LS2_Detectors.
+`--YaxisScale` |bool | If YaxisScale option is used the Y axis is plotted in kUnit. Default is: False.
+`--SetErrX  ` | bool| Set Error option true (for X axis).
+`--SetErrY` | bool | Set Error option true (for Y axis).
+`--SelectColumnErrX` |int | If SetErrX=True choose the Column number for XError, Default is: 0.
+`--SelectColumnErrY` |int | If SetErrY=True choose the Column number for YError, Default is: 0.
+`--PlotLineSize` |float| Set Plot Line Size (default is: 1).
+`--PlotLineStyle` |float| Set Plot Line Style (default is: 1).
+`--PlotMarkerSize` |float| Set Plot Marker Size (default is: 0.8).
+`--Fit ` |bool |If the Fit option is used the header parameters for the fit are created .
+`--FitFormula` | string | `TFormula` to be given to a `ROOT::TF1` object.  Note the syntax and full complexity of inputs expected/available in `ROOT` works! e.g. `[0]*x^2+[1]` or `gaus(0)+pol2(3)" or "[1]*TMath::Erf(x) + [2]` are all supported. Default is [0].
+`--FitOption` | string | The [Fit Method](https://root.cern.ch/root/htmldoc/guides/users-guide/FittingHistograms.html#the-fit-method) to be used for fitting. Default is: R (Use the range specified in the function range)
+`--FitParamIGuess`  | Comma separated list of strings | Initial values for the parameters defined in the `Fit_Formula` field.  Note the order in which the parameters are given in the `Fit_Formula` field should match the order listed here. Explicitly for the `Fit_Formula = '[0]*x^2+[1]*x+[2]'` the initial guess of `4,5,6` would mean `[0] = 4`, `[1] = 5`, and `[2] = 6`.  Additionally, both numeric values and expressions formed from the above supported keywords can be used.  E.g. for a `Fit_Formula = 'gaus(0)'` the initial guess of `12.3, MEAN, 2.*SIGMA+23.5` can be assigned.
+`--FitRange` | Comma separated list of strings | Set the Fit range, default is 0,1000.
+`--FitPerform` | string| If true (default) perform a fit to the TObject defined in the [BEGIN\_PLOT] header this [BEGIN\_FIT] header is found in.
+`--FitLineSize` |float|Set the Fit Line Size, default is: 1 .
+`--FitLineStyle` |float|Set the Fit Line Style, default is: 1 .
```

Comment:
Lines 522-570, this table needs to be re-thought, you are repeating information and in some cases it does not match what is placed elsewhere in the `README.md`: Sections [4.e.iv.I](https://github.com/bdorney/CMS_GEM_Analysis_Framework/#4eivi-header-parameters---canvas), [4.e.iv.II](https://github.com/bdorney/CMS_GEM_Analysis_Framework/#4eivii-header-parameters---plot), and [4.e.iv.III](https://github.com/bdorney/CMS_GEM_Analysis_Framework/#4eiviii-header-parameters---fit).

Rather than repeating the information elsewhere the goal here should be to try to tell the user what input format is expected here and which `FIELD_NAME` each option argument maps to in the aforementioned sections.  This will improve clarity and reduce the chance that conflicting information between this section and section 4.e.iv and subsections are.